libraries/redcore/joomla/image/image.php

```diff
@@ -697,14 +697,10 @@ public function resize($width, $height, $createNew = true, $scaleMethod = self::
 			// Set the transparent color values for the new image.
 			imagecolortransparent($handle, $color);
 			imagefill($handle, 0, 0, $color);
-
-			imagecopyresized($handle, $this->handle, $offset->x, $offset->y, 0, 0, $dimensions->width, $dimensions->height, $this->getWidth(), $this->getHeight());
-		}
-		else
-		{
-			imagecopyresampled($handle, $this->handle, $offset->x, $offset->y, 0, 0, $dimensions->width, $dimensions->height, $this->getWidth(), $this->getHeight());
 		}
 
+		imagecopyresampled($handle, $this->handle, $offset->x, $offset->y, 0, 0, $dimensions->width, $dimensions->height, $this->getWidth(), $this->getHeight());
```

Comment:
@phproberto mentioning you in case you can consider it for JImage in Joomla. I have tested at https://redweb.atlassian.net/browse/RSBTB-2289 and works very nice.